statefun-flink/statefun-flink-distribution/src/main/resources/META-INF/NOTICE

```diff
@@ -0,0 +1,40 @@
+statefun-flink-distribution
+Copyright 2014-2020 The Apache Software Foundation
+
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
+
+This project bundles the following dependencies from (or other subprojects of) the owning top-level project, Apache Flink,
+under the Apache Software License 2.0 (http://www.apache.org/licenses/LICENSE-2.0.txt)
```

Comment:
@zentol do you think we actually need this whole section regarding bundled `flink` / `flink-shaded` dependencies?

Reply:
If the shade-plugin is configured correctly then the Flink entries will be added automatically upon inclusion.

You want to add this transformer:
```
<transformer implementation="org.apache.maven.plugins.shade.resource.ApacheNoticeResourceTransformer">
	<projectName>Apache Flink-statefun</projectName>
	<encoding>UTF-8</encoding>
</transformer>
```